.github/scripts/run-linux-poweroff.py

```diff
@@ -22,7 +19,7 @@ def run_w_timeout(workload, timeout):
             """
             log_tail_length = 100
             # rename runfarm tag with a unique tag based on the ci workflow
-            with prefix(f"export FIRESIM_RUNFARM_PREFIX={ci_workflow_run_id}"):
+            with prefix(f"export FIRESIM_RUNFARM_PREFIX={ci_workflow_run_id}-default"):
```

Comment:
Maybe this would make more sense if I made this automatically determined based off the file name + CI ID so that you don't have to make up a postfix? The postfix is needed so that the run farm name doesn't collide between the two different `run-linux-*.py` scripts.

Reply:
Ok.  Yeah, it would be good to make it use `__file__` in the runfarm prefix to make it clearer what you're doing.